py3-pip.advisories.yaml

```diff
@@ -9,6 +9,11 @@ advisories:
       - CVE-2018-20225
       - GHSA-7p5p-7qq5-cc86
     events:
+      - timestamp: 2024-06-20T14:20:25Z
+        type: false-positive-determination
+        data:
+          type: vulnerability-record-analysis-contested
+          note: This vulnerability is disputed, and the consensus in the security community is that this is intended behavior, not a security flaw.
```

Comment:
should be added after the detection event

Reply:
@cpanato done!